### ISSUE TITLE
Events not showing on Month View

### ISSUE THREAD

**Issue by CloakingTorin** [MMM-CalendarExt2/MMM-CalendarExt2#115]
Hi I'm pretty new to MM and javascript for that matter. I have been trying to configure a Month view to simply show some holidays but the events aren't showing up. I know the ics calendar url works because I can display the info on a weekly view. Can someone please point out what I'm doing wrong. Below is a screenshot ran with my current configuration.
![2021-03-05-133156_1920x1080_scrot](https://user-images.githubusercontent.com/80125374/110165031-dd497c80-7db7-11eb-8891-9bc1a27dc751.png)
My Configuration is shown below
```js
* Magic Mirror Config Sample
 *
 * By Michael Teeuw https://michaelteeuw.nl
 * MIT Licensed.
 *
 * For more information on how you can configure this file
 * See https://github.com/MichMich/MagicMirror#configuration
 *
 */

var config = {
	address: "localhost", 	// Address to listen on, can be:
							// - "localhost", "127.0.0.1", "::1" to listen on loopback interface
							// - another specific IPv4/6 to listen on a specific interface
							// - "0.0.0.0", "::" to listen on any interface
							// Default, when address config is left out or empty, is "localhost"
	port: 8080,
	basePath: "/", 	// The URL path where MagicMirror is hosted. If you are using a Reverse proxy
					// you must set the sub path here. basePath must end with a /
	ipWhitelist: ["127.0.0.1", "::ffff:127.0.0.1", "::1"], 	// Set [] to allow all IP addresses
															// or add a specific IPv4 of 192.168.1.5 :
															// ["127.0.0.1", "::ffff:127.0.0.1", "::1", "::ffff:192.168.1.5"],
															// or IPv4 range of 192.168.3.0 --> 192.168.3.15 use CIDR format :
															// ["127.0.0.1", "::ffff:127.0.0.1", "::1", "::ffff:192.168.3.0/28"],

	useHttps: false, 		// Support HTTPS or not, default "false" will use HTTP
	httpsPrivateKey: "", 	// HTTPS private key path, only require when useHttps is true
	httpsCertificate: "", 	// HTTPS Certificate path, only require when useHttps is true

	language: "en",
	logLevel: ["INFO", "LOG", "WARN", "ERROR"], // Add "DEBUG" for even more logging
	timeFormat: 24,
	units: "imperial",
	// serverOnly:  true/false/"local" ,
	// local for armv6l processors, default
	//   starts serveronly and then starts chrome browser
	// false, default for all NON-armv6l devices
	// true, force serveronly mode, because you want to.. no UI on this device

	modules: [
		{
			module: "clock",
			position: "bottom_right"
		},
{
  module: 'MMM-CalendarExt2',
  config: {
	  
    calendars : [
	{
		name: "Holidays",
		classname: "holiday",
		url: "https://www.thunderbird.net/media/caldata/USHolidays.ics",
	},
    ],
    views: [
      {
		name: "UpComing Holidays",
        mode: "month",
        position: "top_left", 
        calendars: ["Holidays"], 
      },
         {
		name: "Holiday",
        mode: "weekly",
        position: "top_right", 
        calendars: ["Holidays"], 
      },
      
    ],
    
    scenes: [
      {
        name: "DEFAULT",
      },
    ],
  },
},	
```
Thanks

edit by klaernie: add code fencing around config

**Comment by jokajak** [comment]
I had a similar issue where my calendars weren't populating because it failed to parse the US Holidays calendar that's part of the example. I think it's related to https://github.com/MMM-CalendarExt2/MMM-CalendarExt2/pull/87 but I don't think that PR is quite the right fix.

**Comment by CloakingTorin** [comment]
@jokajak Thanks but I think I figured out the issue. I think it is the formatting of the "https://www.thunderbird.net/media/caldata/USHolidays.ics" file because as soon as I replaced it with a google basic calendar "https://calendar.google.com/calendar/ical/en.usa%23holiday%40group.v.calendar.google.com/public/basic.ics" it seems to work perfectly. I'm not familiar with the correct format of the ics files so I haven't investigated it. Maybe in the future.
